### DISCUSSION TITLE
How to disable slide open animation for Neotree?

### DISCUSSION THREAD

**Discussion by gwenwindflower** [LazyVim/LazyVim#2601] [Q&A] [answered]
I like animations for everything except windows sliding open and closed, I've got everything disabled I think except Neotree is still doing a slide open (it closes instantly though).

```lua
-- in my mini.animate config
  local animate = require("mini.animate")
  return {
    open = {
      enable = false,
    },
    close = {
      enable = false,
    },
    resize = {
      enable = false,
    },
```
anybody know what i'm missing? i've looked at Neotree's settings and edgy.nvim for perhaps some animations stuff there but not seeing anything.

**Comment by hoarse-boy** [comment, marked as answer]
hello @gwenwindflower, i had the same issue in the past. if my memory is correct, i think the animation is caused by edgy.nvim. maybe you can try to remove it. but if you still want to use edgy.nvim you can check my [config](https://github.com/jourdy-mago/nvim/blob/main/lua/plugins/edgy-nvim.lua).

**Comment by gwenwindflower** [comment]
It was indeed edgy.nvim -- clearing out my `left` config (I left Neotest results pane in there for now) took care of it, and I already had the Git and Buffer trees disabled so no big loss. Thanks!

**Comment by gwenwindflower** [comment]
It was indeed edgy.nvim -- clearing out my `left` config (I left Neotest results pane in there for now) took care of it, and I already had the Git and Buffer trees disabled so no big loss. Thanks!
